.gitignore

```diff
@@ -1,2 +1,4 @@
 .idea
-out
\ No newline at end of file
+out
+in/example.net.xml
+in/example.stat.xml
```

Comment:
Why are they ignored when you have just added them?

Reply:
Ignoring files that are already a part of the repo doesn't remove them, any further changes are just ignored

Reply:
Because those examples will be changed everytime we run `generateNet`. As I said in the description, further changes to the example are ignored per default. If we want to update the examples we just force include them

Reply:
Wait. That doesn't work as I want it to. I will remove it from the gitignore then.